package.json

```diff
@@ -29,8 +29,7 @@
 		"@emotion/babel-plugin": "^11.12.0",
 		"@wordpress/base-styles": "^5.7.0",
 		"@wordpress/browserslist-config": "^6.7.0",
-		"@wordpress/element": "^6.7.0",
```

Comment:
React components on the admin and sidebar are depending on `@wordpress/element`.

Reply:
Thanks for the details. Builds work successfully!